contrib/mkimage-arch.sh

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env bash
 # Generate a minimal filesystem for archlinux and load it into the local
 # docker as "archlinux"
-# requires root
 set -e
 
+hash sudo &>/dev/null || {
+	echo "Could not find sudo. Run pacman -S sudo"
+	exit 1
+}
+
+[ $EUID -eq 0 ] || exec sudo $0
```

Comment:
Shouldn't whether or not we have `sudo` installed only matter if we're not already root?

ie:
```bash
if [ $EUID -ne 0 ]; then
    # if there is no sudo, this will error with a nice message like "sudo: command not found"
    exec sudo "$BASH_SOURCE" "$@"
fi
```